xarray/backends/h5netcdf_.py

```diff
@@ -448,9 +448,37 @@ def open_datatree(
         driver_kwds=None,
         **kwargs,
     ) -> DataTree:
+
+        from xarray.core.datatree import DataTree
+
+        groups_dict = self.open_groups(filename_or_obj, **kwargs)
+
+        return DataTree.from_dict(groups_dict)
+
+    def open_groups(
+        self,
+        filename_or_obj: str | os.PathLike[Any] | BufferedIOBase | AbstractDataStore,
+        *,
+        mask_and_scale=True,
+        decode_times=True,
+        concat_characters=True,
+        decode_coords=True,
+        drop_variables: str | Iterable[str] | None = None,
+        use_cftime=None,
+        decode_timedelta=None,
+        format=None,
+        group: str | Iterable[str] | Callable | None = None,
+        lock=None,
+        invalid_netcdf=None,
+        phony_dims=None,
+        decode_vlen_strings=True,
+        driver=None,
+        driver_kwds=None,
```

Comment:
These shouldn't be here right? They should all fall under `**kwargs``

Reply:
Or maybe they should be in the specific backend but not in `common.py`?

Reply:
Yeah, so these were added from PR #9199 for adding back the backend specific keyword arguments. I pulled this into my branch after it was merged to main. But they are not in `common.py` , they are consolidated as `**kwargs` in `common.py`.